processor/src/operations/io_ops.rs

```diff
@@ -387,10 +385,9 @@ mod tests {
         // execute the MSTREAM operation
         process.execute_op(Operation::MStream).unwrap();
 
-        // the result the first 8 values should be the result of adding the values on the stack
-        // to the values in memory (each should result in 35), the next 4 values should remain
+        // the first 8 values should contain the values from memory. the next 4 values should remain
         // unchanged, and the address should be incremented by 2 (i.e., 1 -> 3).
-        let stack_values = [35, 35, 35, 35, 35, 35, 35, 35, 4, 3, 2, 1, 3, 101];
+        let stack_values = [23, 24, 25, 26, 27, 28, 29, 30, 4, 3, 2, 1, 3, 101];
```

Comment:
minor nit: It might be better to write these values in terms of `word1` and `word2` arrays for clarity.

Reply:
updated!